src/acq/adc.c

```diff
@@ -655,7 +655,9 @@ static void bl_acq_adc_dma_isr(bl_acq_adc_t *adc, unsigned buffer)
 		if (bl_led_src[bl_led_active].src_mask & adc->config.src_mask) {
```

Comment:
Please change commit message to "Don't flash LED if only one is enabled"

Reply:
+1